cmd/mockery.go

```diff
@@ -145,9 +143,33 @@ func GetRootAppFromViper(v *viper.Viper) (*RootApp, error) {
 	return r, nil
 }
 
+func getFilters(names []string, log zerolog.Logger) ([]*regexp.Regexp, bool) {
```

Comment:
The log should be passed through `ctx`, see some of the other examples in the code of how we do that.